assets/constants.ts

```diff
@@ -78,6 +78,29 @@ export default {
 
   "RTL_LANGUAGE_CODES": ["ar"],
 
+  // Hacky translations keys for classroom, topic, skill, exploration
```

Comment:
This doesn't seem to match the value of the constant, which contains only classroom and topic keys?

Also it should probably be TRANSLATION not TRANSLATIONS.